config/README.md

```diff
@@ -111,6 +111,7 @@ redis:
     - <string> # example "localhost:6379"
   username: <string>
   password: <string>
+  db_index: <int> | default = 0 [optional]
```

Comment:
Hi, can you add a comment that it only works for non clustered redis? Even its "obivous" for a redis poweruser, it might prevent some false issues on the chproxy repo.

Reply:
Fixed